backends/Rebot/src/Connection.h

```diff
@@ -40,7 +40,7 @@ namespace ChimeraTK::Rebot {
    private:
     std::string address_;
     const std::string port_;
-    boost::asio::io_service ioService_;
+    boost::asio::io_context ioService_;
```

Comment:
We should also rename the local variables xxxService into xxxContext.

Reply:
done